src/bidiMapper/bidiProtocolTypes.ts

```diff
@@ -429,3 +429,39 @@ export namespace Session {
     message: string;
   };
 }
+
+// https://w3c.github.io/webdriver-bidi/#module-log
+export namespace Log {
+  export type LogEntry = GenericLogEntry | ConsoleLogEntry | JavascriptLogEntry;
+
+  export type BaseLogEntry = {
+    level: string;
+    text: string | null;
+    timestamp: number;
+    stackTrace?: CommonDataTypes.StackTrace;
+  };
+
+  export type GenericLogEntry = BaseLogEntry & {
+    type: string;
+  };
+
+  export type ConsoleLogEntry = BaseLogEntry & {
+    type: 'console';
+    method: string;
+    real: string;
+    args: [CommonDataTypes.RemoteValue];
+  };
+
+  export type JavascriptLogEntry = BaseLogEntry & {
+    type: 'javascript';
+  };
+
+  export type LogEventAddedInfo = {};
+
+  export type LogEntryAddedEvent = {
+    method: 'log.entryAdded';
+    params: LogEventAddedInfo;
+  };
```

Comment:
It looks like `LogEventAddedInfo` is not needed, and `LogEntry` should be used instead:
```suggestion
  export type LogEntryAddedEvent = {
    method: 'log.entryAdded';
    params: LogEntry;
  };
```

Reply:
Updated. Thanks for the suggestions.